sympy/stats/tests/test_continuous_rv.py

```diff
@@ -42,7 +42,7 @@ def test_single_normal():
     assert variance(Y) == sigma**2
     pdf = density(Y)
     x = Symbol('x')
-    assert (pdf(x) ==
+    assert (pdf(x) == \
```

Comment:
Line continuation isn't needed with brackets. Actually brackets are preferred to line continuation in PEP8